certbot/plugins/standalone.py

```diff
@@ -3,21 +3,30 @@
 import collections
 import logging
 import socket
+from socket import errno as socket_errors  # type: ignore
```

Comment:
Is there an issue tracking the need to use `type: ignore` here? Can we link it?

Reply:
https://github.com/python/typeshed/blob/master/stdlib/2and3/socket.pyi